### DISCUSSION TITLE
Status of v4

### DISCUSSION THREAD

**Discussion by inoyakaigor** [ant-design/ant-design#40170] [Q&A] [answered]
I cannot find any mentions about current v4 status. Is this deprecated or still active? Or will be available only critical fixes?

**Comment by Wxh16144** [comment, marked as answer]
> v4 will continue to be maintained for 1 year, and patches will still be released for bugs, but no new Feature Requests will be accepted after that. The deadline is the end of 2023.

Is that what you want to know? 
learn more: https://github.com/ant-design/ant-design/issues/38671#issue-1454512042
